### PR TITLE
Update packages versions & version UserOwnsData framework to 4.6.1 (in tile and dashboard)

### PR DESCRIPTION
**Pull request by OfirGordon**
1. Updated for Newtonsoft.json and Microsoft.Rest.ClientRuntime packages versions.
2. Updated .Net framework of UserOwnsData app for tile and dashboard embed to .Net framework 4.6.1 (missed those 2 solutions in last update of .Net frameworks)

### CODE REVIEW

File: App Owns Data/PowerBIEmbedded_AppOwnsData/PowerBIEmbedded_AppOwnsData.csproj [hunk 1]
```diff
@@ -64,17 +64,19 @@
...
     </Reference>
     <Reference Include="System" />
     <Reference Include="System.Data" />
     <Reference Include="System.Data.DataSetExtensions" />
     <Reference Include="System.Drawing" />
     <Reference Include="System.Net" />
-    <Reference Include="System.Runtime" />
-    <Reference Include="System.Runtime.Serialization" />
+    <!--Reference Include="System.Runtime" />
+    <Reference Include="System.Runtime.Serialization" -->
```
Comment (MahirDiab) [inline App Owns Data/PowerBIEmbedded_AppOwnsData/PowerBIEmbedded_AppOwnsData.csproj]: why is this commented out?
please un-comment or remove if not needed

Comment (OfirGordon) [reply App Owns Data/PowerBIEmbedded_AppOwnsData/PowerBIEmbedded_AppOwnsData.csproj]: I checked, this is unnecessary, removed

File: User Owns Data/integrate-dashboard-web-app/PBIWebApp/PBIWebApp.csproj [hunk 2]
```diff
@@ -20,8 +20,8 @@
     <IISExpressWindowsAuthentication />
     <IISExpressUseClassicPipelineMode />
     <UseGlobalApplicationHostFile />
-    <TargetFrameworkProfile />
-    <Use64BitIISExpress />
+    <!--TargetFrameworkProfile />
+    <Use64BitIISExpress -->
```
[the remark this answers is not in the document]
Comment (OfirGordon) [reply User Owns Data/integrate-dashboard-web-app/PBIWebApp/PBIWebApp.csproj]: I checked, this is unnecessary, removed

File: User Owns Data/integrate-dashboard-web-app/PBIWebApp/Properties/Settings.settings [hunk 3]
```diff
@@ -7,8 +7,8 @@
     </Setting>
     <Setting Name="ClientSecret" Type="System.String" Scope="Application">
       <Value Profile="(Default)">{Enter your app SecretKey}</Value>
-    </Setting>
-    <Setting Name="AADAuthoritySignInUri" Type="System.String" Scope="Application">
+    </Setting>
```
Comment (MahirDiab) [inline User Owns Data/integrate-dashboard-web-app/PBIWebApp/Properties/Settings.settings]: remove extra spaces

File: User Owns Data/integrate-report-web-app/PBIWebApp/packages.config [hunk 4]
```diff
@@ -14,10 +14,10 @@
...
   <package id="Microsoft.PowerBI.Api" version="2.0.12" targetFramework="net45" />
   <package id="Microsoft.PowerBI.JavaScript" version="2.5.1" targetFramework="net45" />
-  <package id="Microsoft.Rest.ClientRuntime" version="2.0.1" targetFramework="net45" />
+  <package id="Microsoft.Rest.ClientRuntime" version="2.3.18" targetFramework="net461" />
   <package id="Microsoft.Web.Infrastructure" version="1.0.0.0" targetFramework="net45" />
   <package id="Modernizr" version="2.6.2" targetFramework="net45" />
-  <package id="Newtonsoft.Json" version="7.0.1" targetFramework="net45" />
+  <package id="Newtonsoft.Json" version="12.0.1" targetFramework="net461" />
   <package id="Respond" version="1.2.0" targetFramework="net45" />
   <package id="WebGrease" version="1.5.2" targetFramework="net45" />
```
Comment (ali-hamud) [inline User Owns Data/integrate-report-web-app/PBIWebApp/packages.config]: What about all other packages with targetFramework="net45"

**Pull request merged**